website/cloudflare.erb

```diff
@@ -106,6 +106,9 @@
             <li<%= sidebar_current("docs-cloudflare-resource-custom-ssl") %>>
               <a href="/docs/providers/cloudflare/r/custom_ssl.html">cloudflare_custom_ssl</a>
             </li>
+            <li<%= sidebar_current("docs-cloudflare-datasource-devices") %>>
```

Comment:
you'll want this up in the data source section ⬆️ https://github.com/cloudflare/terraform-provider-cloudflare/pull/1453/files#diff-92335e11612bda51cfa238d710846cda62d064dfc2dad5ba57141b33065acd39R27